cli/tests/integration/lsp_tests.rs

```diff
@@ -2428,18 +2428,12 @@ fn lsp_code_lens() {
         "end": { "line": 0, "character": 7 }
       },
       "command": {
-        "title": "2 references",
+        "title": "1 reference",
         "command": "deno.showReferences",
         "arguments": [
           "file:///a/file.ts",
           { "line": 0, "character": 6 },
           [{
-            "uri": "file:///a/file.ts",
-            "range": {
-              "start": { "line": 0, "character": 6 },
-              "end": { "line": 0, "character": 7 }
-            }
-          }, {
```

Comment:
Our code lens was displaying as always having at least 1 reference, but it should be 0 when there are 0 references.

Reply:
I don't see a test for 0 reference, could you add it?

Reply:
This is essentially the test for that (the declaration is no longer included).

Reply:
Oh wait, there's a bug in the output for 0 references lol. Let me write a unit test.

Reply:
Oh no, it's there. I misread the code, but let me improve it so it's easier to read.